pyproject.toml

```diff
@@ -1,6 +1,270 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+    "hatchling>=1.2.0"
+]
+
+[project]
+name = "elyra"
+description="Elyra provides AI Centric extensions to JupyterLab"
+maintainers = [
+    {name = "Elyra Maintainers"},
+]
+readme = "README.md"
+requires-python = ">=3.7"
+dependencies = [
+    "autopep8>=1.5.0",
+    "click>=8",  # elyra-ai/elyra#2579
+    "colorama",
+    "deprecation",
+    "entrypoints>=0.3",
+    "jinja2>=3",
+    "jsonschema>=3.2.0,<4.0",  # Cap from kfp
+    "jupyter_core>=4.6.0",
+    "jupyter_client>=6.1.7",
+    "jupyter-packaging>=0.10",
+    "jupyter_server>=1.7.0",
+    "jupyterlab>=3.4.6",  # comment out to use local jupyterlab
+    "jupyterlab-lsp>=3.8.0",  # comment out to use local jupyterlab
+    "jupyterlab-git~=0.32",  # Avoid breaking 1.x changes
+    "jupyter-resource-usage>=0.5.1",
+    "MarkupSafe>=2.1",
+    "minio>=7.0.0",
+    "nbclient>=0.5.1",
+    "nbconvert>=6.4.5",
```

Comment:
Should we bump this floor to `6.5.1` due to recent [dependency bot warnings](https://github.com/elyra-ai/elyra/security/dependabot/76).  Although this warning is relative to the test-requirements file, it should also apply to general installation requirements.

To that point, it looks like [the floor on `jupyter_core` should be increased to `4.11.2`](https://github.com/elyra-ai/elyra/security/dependabot/81) as well.